packages/cli/src/cmds/compare/detectRevisions.ts

```diff
@@ -2,13 +2,17 @@ import { executeCommand } from '../../lib/executeCommand';
 import { prominentStyle } from './ui';
 import { ValidationError } from './ValidationError';
 
+export async function detectRevision(): Promise<string> {
+  return (await executeCommand(`git rev-parse HEAD`)).trim().split('\n')[0];
```

Comment:
Unreleated - but I was just looking at [simple-git](https://www.npmjs.com/package/simple-git). Might simplify things a bit for us.

Reply:
Looks nice, we can migrate to that.